### PR TITLE
ChangeBuilder now has addAll method

### PR DESCRIPTION
**Pull request by yazad3**
### Description:

Added `addAll` methods in `ChangeBuilder`.

### Potential Impact:

None

### Unit Test Approach:

Test checks if addAll behaves the same as if add was called individually for each feature change.

### Test Results:

Test successful.

------

In doubt: [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

File: src/main/java/org/openstreetmap/atlas/geography/atlas/change/ChangeBuilder.java [hunk 1]
```diff
@@ -48,6 +52,42 @@ public synchronized ChangeBuilder add(final FeatureChange featureChange)
...
+     * Iteratively {@link #add(FeatureChange)} all the FeatureChanges.
+     *
+     * @param featureChanges
+     *            - The featureChanges to add.
+     * @return ChangeBuilder - returns itself to allow fluency in calls.
+     */
+    public synchronized ChangeBuilder addAll(final Stream<FeatureChange> featureChanges)
+    {
+        featureChanges.map(this::add).reduce((first, second) -> second);
+
```
Comment (matthieun) [inline src/main/java/org/openstreetmap/atlas/geography/atlas/change/ChangeBuilder.java]: Whitespace

File: src/main/java/org/openstreetmap/atlas/geography/atlas/change/ChangeBuilder.java [hunk 2]
```diff
@@ -48,6 +52,42 @@ public synchronized ChangeBuilder add(final FeatureChange featureChange)
...
+    /**
+     * Iteratively {@link #add(FeatureChange)} all the FeatureChanges.
+     *
+     * @param featureChanges
+     *            - The featureChanges to add.
+     * @return ChangeBuilder - returns itself to allow fluency in calls.
+     */
+    public synchronized ChangeBuilder addAll(final Stream<FeatureChange> featureChanges)
+    {
+        featureChanges.map(this::add).reduce((first, second) -> second);
```
Comment (matthieun) [inline src/main/java/org/openstreetmap/atlas/geography/atlas/change/ChangeBuilder.java]: Why not use forEach here to avoid the reduce?

Comment (yazad3) [reply src/main/java/org/openstreetmap/atlas/geography/atlas/change/ChangeBuilder.java]: Good point.

**Comment by yazad3** [comment]
@matthieun - thanks, review comments addressed.

**Pull request merged**